src/videojs.markers.js

```diff
@@ -231,21 +227,26 @@ type Marker = {
       ) {
         if (overlayIndex !== currentMarkerIndex) {
           overlayIndex = currentMarkerIndex;
-          breakOverlay && breakOverlay.find('.vjs-break-overlay-text').html(setting.breakOverlay.text(marker));
+          if (breakOverlay) breakOverlay.querySelector('.vjs-break-overlay-text').innerHTML = setting.breakOverlay.text(marker);
         }
 
-        breakOverlay && breakOverlay.css('visibility', "visible");
+        if (breakOverlay) breakOverlay.style.visibility = "visible";
       } else {
         overlayIndex = NULL_INDEX;
-        breakOverlay && breakOverlay.css("visibility", "hidden");
+        if (breakOverlay) breakOverlay.style.visibility = "hidden";
```

Comment:
can we keep the original style? `A && ...`
I'm not a big fan of if statement without {}. So you can also do
```
if (A) {
...
}
```